woocommerce/woocommerce
# UI changes for set up payments task

### Changes proposed in this Pull Request:

Closes #32131.

This PR accomplishes the following ACs:
- Split the section into two sections: "Choose a payment provider" [online payments] and "Offline payment methods" [offline payments]
- Change the CTA copy to "Get started"
- Update the payment's icon
- Add a "See more" option at the bottom of the online payments section with a link to https://woocommerce.com/product-category/woocommerce-extensions/payment-gateways/online-payments/
- Removes "Enabled payment gateways" section

If the store is located in a country where WCPayments is eligible but it's not installed
-  Hide the online and offline payment sections under an "Other payment methods" toggle
- Remove the online payment header section

Adds the following event tracking:
- `wcadmin_tasklist_payment_see_more` when "See more" button is clicked
- `wcadmin_tasklist_payment_show_toggle` when "Other payment method" button is clicked

### How to test the changes in this Pull Request:

**Test with eligible WCPay**:
1. In a fresh site, go through the setup wizard with WCPay eligible country but don't install WCpay.
2. Set `woocommerce_show_marketplace_suggestions` to `no`
3. Go to "Set up payments" task
4. Observe only WCPay card is displayed with an "Other payment methods" button
5. Click on "Other payment methods" toggle button
6. Observe the hidden payment methods are shown

<img width="400" alt="image" src="https://user-images.githubusercontent.com/3747241/162680226-77ec0f09-bf74-4c51-85cc-3b9d665d954b.png">


**Test with ineligible WCPay**
1. Set store country to somewhere ineligible for WCPay, such as Mexico.
2. Go to "Set up payments" task
3. Observe the "Choose a payment provider" section and "Offline payment methods" section
4. Observe no "Other payment methods" toggle button

<img width="400" alt="image" src="https://user-images.githubusercontent.com/3747241/162686661-876ddc64-f4eb-4d6b-b8b9-ad270b458724.png">

**Test various UI changes**
1. Go to "Set up payments" task
2. Observe the following:
    - Payment method icons are now smaller
    - "Set up" button next to methods has been changed to "Get started"

**Test new tracks**
1. Set store country to somewhere eligible for WCPay, such as US.
1. Go to "Set up payments" task
2. Open up browser console and run `localStorage.setItem( 'debug', 'wc-admin:tracks' );` to set up tracks debugging
3. Click on "Other payment methods" button
4. Observe `wcadmin_tasklist_payment_show_toggle` track is recorded with `toggle` and `payment_method_count` props
5. Click on "See more" button
6. Observe `wcadmin_tasklist_payment_see_more` track is recorded
7. Observe the "See more" button should go to `https://woocommerce.com/product-category/woocommerce-extensions/payment-gateways/?utm_source=payments_recommendations`


### Changelog entry

Dev: UI changes for set up payments task

### FOR PR REVIEWER ONLY:
